src/sentry/static/sentry/app/components/activity/note/types.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Represents a mentionable user or team.
+ */
+export type Mentionable = {
+  id: string;
+  display: string;
+  email: string;
+};
+
+/**
+ * List of id, display name
+ */
+export type Mentioned = [string, string];
+
+/**
+ * The typings for the react-mentionables library use this type
+ * for their change event so we also use it.
+ */
+export type MentionChangeEvent = {target: {value: string}};
```

Comment:
should we use maybe:
`import {OnChangeHandlerFunc} from 'react-mentions';`
`export type MentionChangeEvent = Parameters<OnChangeHandlerFunc>[0]` ?